templates/stamp/stamp-v2.json

```diff
@@ -692,6 +704,42 @@
         "keyVaultEnabledForVolumeEncryption": true,
         "keyVaultTenantId": "[subscription().subscriptionId]",
 
+        "resourcesToDeploy": {
+            "bootstrap": {
```

Comment:
Can this be compressed with inline conditional.

Reply:
Yeah, I just thought this was cleaner. I can do either though.